apollo-server/composants/utilisateur/Profile.js

```diff
@@ -41,31 +41,47 @@ const Profile = bdd.define(
       type: Sequelize.STRING,
       unique: true,
       validate: {
-        isEmail: true
+        isEmail: {
+          msg: 'Ce champ doit être une adresse e-mail valide.'
+        }
       }
     },
 
     nom: {
       type: Sequelize.STRING,
       allowNull: false,
       validate: {
-        isAlpha: true
+        is: {
+          args: /(?:\w|[\- ])+/gi,
+          msg: 'Ce champ ne peut contenir que des lettres minuscules ou majuscules, des tirets (-) et des espaces.'
+        },
+        len: {
+          args: [ 2, 20 ],
+          msg: 'Ce champ doit contenir entre 2 et 20 caractères.'
+        }
       }
     },
     prenom: {
       type: Sequelize.STRING,
       allowNull: false,
       validate: {
-        isAlpha: true
+        is: {
+          args: /(?:\w|[\- ])+/gi,
+          msg: 'Ce champ ne peut contenir que des lettres minuscules ou majuscules, des tirets (-) et des espaces.'
+        },
+        len: {
+          args: [ 2, 20 ],
```

Comment:
La taille de ce champ me parait un peu limite, surtout pour les noms un peu compliqués. Je propose 30 caractères.
```suggestion
          args: [ 2, 30 ],
```